pkg/pillar/rstats/rstats_n.go

```diff
@@ -0,0 +1,12 @@
+//go:build !rstats
+// +build !rstats
+
+package rstats
+
+// Endpoint is statsd endpoint address, gets replaced at build time.
+var Endpoint = "<ip>:<port>"
+
+// Tag is bucket tag, gets replaced at build time.
+var Tag = "<tag>"
+
+func init() {}
```

Comment:
Why do you need this empty init function?
You get golangcilint: don't use `init` function (gochecknoinits)

Reply:
I thought to keep the look of Y/N files consistent, removed it.